doc/Tex/RAM_SCB_intro.tex

```diff
@@ -1,56 +1,17 @@
-lines passing through the SM equatorial plane at a radial distance of 6.5 
-$R_{E}$.  The two codes work in tandem, with RAM providing anisotropic pressure
-to SCB and SCB returning the self-consistent magnetic field through which RAM 
-plasma is advected.
-
-RAM-SCB has grown from a research-grade code with limited options and static 
-magnetic field (RAM) to a rich, highly configurable research and operations 
-tool with a multitude of new physics and output products.  This manual provides
-a guide to users who want to learn how to install, configure, and execute 
-RAM-SCB simulations.  While the code is designed to make these steps as
-straight-forward as possible, it is strongly recommended that users review
-the publications listed in the Bibliography to ensure a thorough understanding 
-of the physics included in the model.  Additionally, all users are asked to 
-review the terms of use.
+The \textbf{R}ing current \textbf{A}tmosphere interactions \textbf{M}odel with \textbf{S}elf \textbf{C}onsistent magnetic field (\textbf{B}) is a unique code that combines a kinetic model of ring current plasma with a three dimensional force-balanced model of the terrestrial magnetic field.  The kinetic portion, RAM, solves the kinetic equation to yield the bounce-averaged distribution function as a function of azimuth, radial distance, energy and pitch angle for four ion species ($H^{+}$, $He^{+}$, $N^{+}$, and $O^{+}$) and, optionally, electrons. The code is also set up to easily accept new ion species. The domain is a circle in the Solar-Magnetic (SM) equatorial plane with a configurable radial span (2 to 6.5 $R_{E}$ by default).  It has an energy range of approximately 100\,$eV$ to 500\,$KeV$.  The 3-D force balanced magnetic field model, SCB, balances the $\textbf{J} \times \textbf{B}$ force with the divergence of the general pressure tensor to calculate the magnetic field configuration within its domain.  The domain ranges from near the Earth's surface, where the field is assumed dipolar, to the shell created by field lines passing through the SM equatorial plane at radial distances which overlap the RAM domain.  The two codes work in tandem, with RAM providing anisotropic pressure to SCB and SCB returning the self-consistent magnetic field through which RAM plasma is advected.
+
+RAM-SCB has grown from a research-grade code with limited options and static magnetic field (RAM) to a rich, highly configurable research and operations tool with a multitude of new physics and output products.  This manual providesa guide to users who want to learn how to install, configure, and execute RAM-SCB simulations.  While the code is designed to make these steps asstraight-forward as possible, it is strongly recommended that users review the publications listed in the Bibliography to ensure a thorough understanding of the physics included in the model.  Additionally, all users are asked to review the terms of use.

```

Comment:
Couple of spaces missing in this block of text

Reply:
Just noticed "to the shell created by field lines passing through the SM equatorial plane at radial distances which overlap the RAM domain." The SCB domain isn't defined that way since the grids were decoupled... right?